cmd/horcrux/cmd/signer.go

```diff
@@ -55,34 +54,19 @@ func startSignerCmd() *cobra.Command {
 				return err
 			}
 
-			var (
-				// services to stop on shutdown
-				services []cometservice.Service
-				logger   = cometlog.NewTMLogger(cometlog.NewSyncWriter(os.Stdout)).With("module", "validator")
-			)
-
-			_, err = config.KeyFileExistsSingleSigner()
-			if err != nil {
-				return err
-			}
-
-			logger.Info("CometBFT Validator", "mode", "single-signer",
-				"priv-key", config.Config.PrivValKeyFile, "priv-state-dir", config.StateDir)
+			logger := cometlog.NewTMLogger(cometlog.NewSyncWriter(os.Stdout)).With("module", "validator")
```

Comment:
This should use `cmd.OutOrStdout()` instead of `os.Stdout` -- I think I may have missed at least one earlier occurrence of this.